.travis.yml

```diff
@@ -1,18 +1,37 @@
 language: node_js
+
 node_js:
-- "lts/*"
-before_deploy:
-- test $TRAVIS_TEST_RESULT = 0
-deploy:
-- provider: script    # (If version.txt is updated) - create a new tag and push to Github, update the latest-release branch
-  script: ./create-release.sh
-  on: master
-  # Automatic deploys are enabled in Heroku for this app
-  # Every push to master will deploy a new version of this app. Deploys happen automatically.
-  # Heroku will wait for CI to pass before deploying.
+  - "lts/*"
```

Comment:
this looks to take our test suite from 1 runner to 6?

1. Does this cause a bottleneck?
2. Do we need to pay for this?

Reply:
That's correct – it'll run the tests six times for each combination of node version and operating system. It will attempt to run these in parallel if there are test runners free.

It _could_ create a bottleneck – if there are other repos already building for the organisation, it will take significantly longer than if there was only one build to run, as they won't be able to run in parallel. On a good day, they'll all run concurrently and it'll take marginally longer than just running once.

It won't cost any extra – I _believe_ the limit for the free account is that we have six runners for the organisation.

Personally I think the overhead is worth it for the increased confidence that the kit will work for Windows users. We can also review it if it becomes problematic.

Reply:
👍 
I'd feel more comfortable if we had a plan to have a paid Travis tier but let's see how it goes.